### PR TITLE
Add stream_folders path to OpenEphysBinaryRecordingExtractor

### PR DESCRIPTION
**Pull request by LiesDec**
For other analyses I require the direct data path of the raw data of on Binary Open Ephys recording (continuous.dat). 
For easy acces I propose to add it to the _kwargs property of the OpenEphysBinaryRecordingExtractor. 
Thanks! :)

### CODE REVIEW

File: src/spikeinterface/extractors/neoextractors/openephys.py [hunk 1]
```diff
@@ -242,6 +242,7 @@ def __init__(
                 load_sync_channel=load_sync_channel,
                 load_sync_timestamps=load_sync_timestamps,
                 experiment_names=experiment_names,
+                stream_folder=stream_folder,
```
Comment (alejoe91) [inline src/spikeinterface/extractors/neoextractors/openephys.py:245]:
```suggestion
```
Hi @LiesDec 

The `self._kwargs` contains only the argument that are passed to the init. If you want to stor the `stream_folder`, you can store it in a `self._stream_folder` variable instead

**Review by alejoe91 — CHANGES_REQUESTED** [review]
See comment ;)

File: src/spikeinterface/extractors/neoextractors/openephys.py (hunk 1 above)
Comment (LiesDec) [reply src/spikeinterface/extractors/neoextractors/openephys.py]: I changed it in a new commit :)

File: src/spikeinterface/extractors/neoextractors/openephys.py [hunk 2]
```diff
@@ -244,6 +244,7 @@ def __init__(
                 experiment_names=experiment_names,
             )
         )
+        self._stream_folder = stream_folder
```
Comment (alejoe91) [inline src/spikeinterface/extractors/neoextractors/openephys.py:247]: Can you move this before that `self._kwargs` logic? We try to keep that at the end of the init so we know where to look for it!

Comment (LiesDec) [reply src/spikeinterface/extractors/neoextractors/openephys.py]: Of course :) Good to learn these conventions, thanks for your patience.

**Review by alejoe91 — CHANGES_REQUESTED** [review]
@LiesDec 

I noticed that `stream_folder` is not defined if `load_timestamps=False`.
See: https://github.com/SpikeInterface/spikeinterface/blob/41a75e2502a00f1f8b0f958b3433ad1787c40b87/src/spikeinterface/extractors/neoextractors/openephys.py#L223

Can you move that line before the if?

**Comment by alejoe91** [comment]
@LiesDec actually the stream folders should be a list, because there can be multiple recording segments (`recording*` folders in Open Ephys) associated to the same stream. 

I'll push a patch

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**